rpc_new/util.ts

```diff
@@ -0,0 +1,12 @@
+// TODO: expose these utils from `util/mod.ts`
```

Comment:
Any reason not to do this now?

Reply:
It'll break effects that use an existing `Listener`/`CreateListener`. Will do this in a follow-up PR.